go.mod

```diff
@@ -2,6 +2,25 @@ module sigs.k8s.io/oci-proxy
 
 go 1.17
 
-require k8s.io/klog/v2 v2.40.1
+require (
+	github.com/google/go-containerregistry v0.8.0
```

Comment:
let's move the test tool to it's own module, so we can clearly keep imports minimal in the main binary and not require downloading + installing test-only deps during building.

Reply:
(or, perhaps rework entirely 😅 https://github.com/kubernetes-sigs/oci-proxy/pull/32#pullrequestreview-925591249)